lms/djangoapps/sneakpeek_deeplink/tests.py

```diff
@@ -63,6 +63,13 @@ def setUp(self):
             enrollment_start=now - month2,
             enrollment_end=now - month)
         self.closed_course.save()
+        self.course_to_delete = CourseFactory.create(
+            org='deleted',
+            number='course',
+            run='run',
+            enrollment_start=now - month,
+            enrollment_end=now + month)
```

Comment:
Add a trailing comma after the final `kwarg` and drop the trailing paren down to a newline;
it should align with the start of this statement.